app/src/main/admin/components/AdminRouter.tsx

```diff
@@ -30,8 +31,9 @@ export const AdminRouter : React.StatelessComponent<AdminRouterProps> = (props:
         <Route exact path="/modelling-groups/:groupId/" component={ModellingGroupDetailsPage}/>
         <Route exact path="/modelling-groups/:groupId/admin/" component={ModellingGroupMembersPage}/>
         <Route exact path="/touchstones/" component={TouchstoneListPage}/>
+        <Route exact path="/touchstones/:touchstoneVersionId/" component={TouchstoneVersionPage}/>
```

Comment:
So, in VIMC-1885, I have added `/touchstones/:touchstoneId`. Maybe the URL scheme for versions of touchstone should be: `/touchstones/:touchstoneId/:versionId`, e.g. `/touchstones/201810gavi/3/`?

Reply:
ah yes, makes sense